lib/run/summary.js

```diff
@@ -12,8 +12,18 @@ var _ = require('lodash'),
  */
 RunSummary = function RunSummary (emitter, options) {
     // keep a copy of this instance since, we need to refer to this from various events
-    var summary = this;
-
+    var summary = this,
+        onlyCLIReporterEnabled,
+        skipExecutions;
+
+    // Need to do add this check for unit-tests when no options are passed
+    if (options) {
+        onlyCLIReporterEnabled = (_.isArray(options.reporters) &&
+            options.reporters.length === 1 &&
+            options.reporters.includes('cli'));
+        // skipExecutions in case the option is specified, or only cli is specified as reporter
+        skipExecutions = _.get(options, 'skipExecutions') || onlyCLIReporterEnabled;
```

Comment:
In case `skipExecutions` is specified, we can avoid computing `onlyCLIReporterEnabled`.